README.md

```diff
@@ -26,7 +26,7 @@ Bicep compiles down to standard ARM Template JSON files, which means the ARM JSO
 
 ## Get started with Bicep
 
-To get going with bicep, start by [installing the tooling](./docs/installing.md).
+To get going with bicep, start by [installing the tooling](./docs/installing.md) or use the [VSCode Devcontainer/Codespaces](https://github.com/Azure/vscode-remote-try-bicep) repo to get a preconfigured environment.
```

Comment:
Would like to keep the main install path as simple as possible. Can we instead add this not in the same sentence as the bicep playground a little bit further down in the doc?

> Alternatively, you can try the Bicep Playground or use the VS Code Devcontainer/Codespaces repo to get a preconfigured environment.

Reply:
Make sense, should be all set now.